x/storage/types/crosschain.go

```diff
@@ -1286,3 +1386,114 @@ func (p UpdateGroupMemberAckPackage) MustSerialize() []byte {
 	}
 	return encodedBytes
 }
+
+type CreatePolicySynPackage struct {
+	Operator  sdk.AccAddress
+	Data      []byte
+	ExtraData []byte
+}
+
+type CreatePolicySynPackageStruct struct {
+	Operator  common.Address
+	Data      []byte
+	ExtraData []byte
+}
+
+var (
+	createPolicySynPackageType, _ = abi.NewType("tuple", "", []abi.ArgumentMarshaling{
+		{Name: "Operator", Type: "address"},
+		{Name: "Data", Type: "bytes"},
+		{Name: "ExtraData", Type: "bytes"},
+	})
+
+	createPolicySynPackageArgs = abi.Arguments{
+		{Type: createPolicySynPackageType},
+	}
+)
+
+func (p CreatePolicySynPackage) ValidateBasic() error {
+	if p.Operator.Empty() {
+		return sdkerrors.ErrInvalidAddress
+	}
+
+	var policy permtypes.Policy
+	err := json.Unmarshal(p.Data, &policy)
```

Comment:
why `json` is used? I hope rlp or even protobuf is accepted.

Reply:
once you get the policy, is any verification agains the policy?

Reply:
> once you get the policy, is any verification agains the policy?

added verification